src/main/scala/is/hail/utils/richUtils/Implicits.scala

```diff
@@ -98,3 +99,106 @@ trait Implicits {
 
   implicit def toRichJValue(jv: JValue): RichJValue = new RichJValue(jv)
 }
+
+object NumericJavaLang {
```

Comment:
this is the wrong place for this object, I think. Put this in a standalone class in the richUtils package and add `implicit val`s in `Implicits.scala`